src/descriptor.rs

```diff
@@ -148,10 +150,143 @@ impl Descriptor {
     }
 
     /// Constructs a new [`Descriptor`] from a byte [`Vec`].
+    ///
+    /// Users are advised to use the `new_*` methods whenever possible.
     pub fn from_vec(bytes: Vec<u8>) -> Result<Self, DescriptorError> {
         Self::from_bytes(&bytes)
     }
 
+    /// Constructs a new [`Descriptor`] from an `OP_RETURN` payload.
+    ///
+    /// The payload is expected to be at most 80 bytes.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use bitcoin_bosd::{Descriptor, DescriptorType};
+    /// let payload = b"hello world";
+    /// let desc = Descriptor::new_op_return(payload).expect("valid payload under 80 bytes");
```

Comment:
Looks like this needs to be updated too.

Reply:
Squashed into d5daf8f2d68eb8097dcbab5a60eae25191ca589e